app/services/utils/pdf_generator.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require_relative '../../views/helpers/line_break_helper'
```

Comment:
I suggest to add the helper_folder in config.eager_load_paths (in /config/application.rb)